tests/test_data/parse_output/output_Test_21.iedb.parsed.top.tsv

```diff
@@ -1,24 +1,24 @@
-22	22550509	22550510	T	G	ENST00000390285	ENSG00000211640	missense	S/A	63	IGLV6-57	HLA-A*02:01	9	7	RPGSAPTTV	RPGSSPTTV	16036.52	16523.8	1.030	SMM	16271.59									21118.0	20554.0	16523.8	16036.52	16960.16	16271.59
-22	37966274	37966275	C	G	ENST00000215886	ENSG00000100079	missense	E/Q	132	LGALS2	HLA-A*02:01	9	13	NMSSFKLKQ	NMSSFKLKE	20458.0	25689.0	1.256	NetMHC	51573.9									25689.0	20458.0	91224.19	53840.61	72016.17	51573.9
-22	50555769	50555770	G	A	ENST00000262794	ENSG00000073146	missense	A/T	482	MOV10L1	HLA-A*02:01	9	3	AKTTVVVTT	AKTTVVVTA	23080.0	20335.0	0.881	NetMHC	83258.77									20335.0	23080.0	24553.31	88533.98	23196.9	83258.77
-22	29886116	29886117	C	A	ENST00000310624	ENSG00000100285	missense	P/T	830	NEFH	HLA-A*02:01	9	6	KEEEKTQEV	KEEEKPQEV	12836.01	15150.61	1.180	SMMPMBEC	16638.34									22210.0	19760.0	19458.53	16638.34	15150.61	12836.01
-22	50615580	50615581	C	T	ENST00000395842	ENSG00000073150	missense	S/F	147	PANX2	HLA-A*02:01	9	4	LGWEFLAFT	LGWEFLAST	1128.31	2001.84	1.774	SMMPMBEC	1352.42									4505.0	2647.0	2296.73	1352.42	2001.84	1128.31
-22	46653595	46653596	G	A	ENST00000253255	ENSG00000130943	missense	T/I	1875	PKDREJ	HLA-A*02:01	9	3	YYSYGLLHI	YYSYGLLHT	1037.79	5573.27	5.370	SMM	1053.0									10178.0	2944.0	5573.27	1037.79	5707.29	1053.0
-22	50869713	50869714	C	A	ENST00000395741	ENSG00000100239	missense	S/Y	414	PPP6R2	HLA-A*02:01	9	7	EASGYESRV	EASGSESRV	17677.87	58267.99	3.296	SMMPMBEC	17787.3									25970.0	20815.0	55476.62	17787.3	58267.99	17677.87
-6	41754573	41754573	C	CCTT	ENST00000458694	ENSG00000124593	inframe_ins	-/L	287-288	PRICKLE4	HLA-A*02:01	9	5	TLSRTLLLA	TLSRTLLAA	402.82	526.15	1.306	SMM	430.95									486.0	451.0	526.15	402.82	564.2	430.95
-4	40434704	40434725	AGCGGCTGCGGCGGCTGCGGCC	A	ENST00000381793	ENSG00000163694	inframe_del	AAAAAAAA/A	495-502	RBM47	HLA-A*02:01	9	4	SAAAAAAAV	SAAAAAAAA	1106.9	5509.47	4.977	SMM	1170.66									9700.0	1235.0	5509.47	1106.9	5800.03	1170.66
-22	26936775	26936776	G	T	ENST00000338754	ENSG00000128294	missense	P/H	274	TPST2	HLA-A*02:01	9	6	DLIGKHGGV	DLIGKPGGV	2037.56	1079.22	0.530	SMM	2230.64									11952.0	16231.0	1079.22	2037.56	1102.63	2230.64
-22	38119219	38119220	GA	G	ENST00000406386	ENSG00000100106	FS		219	TRIOBP	HLA-A*02:01	9	1596	SQWKKHWFV	NA	10.0	NA	NA	NetMHC	44.99									NA	10.0	NA	44.99	NA	52.53
-22	50682229	50682230	T	C	ENST00000248846	ENSG00000128159	missense	H/R	220	TUBGCP6	HLA-A*02:01	9	4	SLFGALVRS	SLFGALVHS	722.95	565.08	0.782	SMM	728.0									457.0	728.0	565.08	722.95	589.43	771.67
-22	18644672	18644673	C	T	ENST00000215794	ENSG00000184979	missense	A/V	124	USP18	HLA-A*02:01	9	6	RPLELVYCL	RPLELAYCL	863.83	1804.8	2.089	SMMPMBEC	972.99									9916.0	5180.0	1995.77	972.99	1804.8	863.83
+22	41895790	41895791	C	A	ENST00000216254	ENSG00000100412	missense	A/E	33	ACO2	HLA-A*02:01	9	9	KVEMSHFEP	KVAMSHFEP	23862.0	18095.0	0.758	NetMHC	46463.29	38.5035	35.4301	NA	NA	NA	NA	NA	NA	18095.0	23862.0	7483.59	46463.29	8578.87	47362.0

```

Comment:
Does this change have other effects besides setting `NA` on things?  I'm surprised to see new numbers appearing in the output.